cmd/ls/ls.go

```diff
@@ -43,12 +45,30 @@ func formatFileName(name string) string {
 	return name
 }
 
+func lookupIds(uid, gid string) (string, string) {
+	u, err := user.LookupId(uid)
+	if err != nil {
+		fatal(err)
```

Comment:
I'm a little torn if the better behavior is to call Exit directly or just panic. The advantage of panicking is that it will give a full stack for debug + it will execute any cleanups performed by possible defers (if not now, there may be on the future, even if the tool remain small).